src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result = initialValue;
```

Comment:
Just to be consistent with reduce naming
```suggestion
    let accumulator = initialValue;
```